homeassistant/components/imeon_inverter/strings.json

```diff
@@ -0,0 +1,186 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add Imeon inverter",
+        "data": {
+          "address": "[%key:common::config_flow::data::url%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "address": "The address of the inverter",
```

Comment:
What are we looking for address? IP? Why does the label say URL? Do we support getting a `http://123.123.123.123`?